src/app/services/communibee-backend/subregion/subregion.ts

```diff
@@ -1,9 +1,10 @@
-import BackendModel from '../backend-model';
-import { Regions, RegionsModel } from '../region/region';
-
-export interface SubRegions extends Regions {
-  region: RegionsModel;
-}
-
-export interface SubRegionsModel extends BackendModel, SubRegions {
-}
+import BackendModel from '../backend-model';
+import { Regions, RegionsModel } from '../region/region';
+
+export interface SubRegions extends Regions {
```

Comment:
Decide if to keep as extends Regions and then delete the name property because Regions has a name, 
or remove the extends.

Reply:
Removed inheritance.